res/controllers/Pioneer-DDJ-SX-scripts.js

```diff
@@ -64,38 +61,106 @@ PioneerDDJSX.autoDJShuffleAfterSkip = false; //default: false
 PioneerDDJSX.jumpPreviewEnabled = true; //default: true
 PioneerDDJSX.jumpPreviewPosition = 0.3; //default: 0.3
 
-// If true, LoadSamplerBank file dialog will show up at running init.
-PioneerDDJSX.samplerShowLoadDialog = true; //default: true
 // If true, pad press in SAMPLER-PAD-MODE repeatedly causes sampler to play 
 // loaded track from cue-point, else it causes to play loaded track from the beginning.
 PioneerDDJSX.samplerCueGotoAndPlay = false; //default: false
 
 // If true, PFL / Cue (headphone) is being activated by loading a track into certain deck.
 PioneerDDJSX.autoPFL = true; //default: true
 
-// If true, new in Mixxx 2.1 introduced library controls will be used (buggy),
+// If true, new in Mixxx 2.1 introduced library controls will be used,
 // else old playlist controls will be used.
 PioneerDDJSX.useNewLibraryControls = false; //default: false
 
+// If true, SHIFT has to be pressed to activate needle search control
+PioneerDDJSX.needleSearchShiftEnable = false; //default: false
```

Comment:
Or you could move them to the line above in case someone forgot what the default was after changing it and wants to change it back.